src/helpers.ts

```diff
@@ -310,10 +310,10 @@ export function createPoolTransaction(
 
   pool.consumePrice = poolTx.consumePrice
   pool.spotPrice = poolTx.spotPrice
-  const lockedValue = pool.lockedValue
-  pool.lockedValue = pool.oceanReserve + (pool.datatokenReserve * pool.spotPrice)
-  let factory = PoolFactory.load('1')
-  factory.totalLockedValue = factory.totalLockedValue - lockedValue + pool.lockedValue
+  pool.lockedValue = pool.oceanReserve + pool.datatokenReserve * pool.spotPrice
+  const factory = PoolFactory.load('1')
+  factory.totalLockedValue =
+    factory.totalLockedValue - pool.lockedValue + pool.lockedValue
```

Comment:
@ssallam did not further touch this but no clue why we are subtracting first and then adding same value again?

Reply:
in main branch it is:

Reply:
maybe that was a typo for some short time

Reply:
it should be adding the new value, and subtracting the old value.

Reply:
perfect thanks! Made the intention more clear in code in 6b304e0